frontend/src/components/EditRecipe/EditRecipeForm.css

```diff
@@ -1,17 +1,24 @@
+textarea, select, input{
```

Comment:
While we're working on this, we should probably get away from styling every `textarea`, `select`, `input`, and `label` on the entire site. Can we change these (and the ones below) to use classes? If there are site-wide styling decisions we want to make, they should probably go somewhere other than this CSS file.